docs/index.rst

```diff
@@ -2,74 +2,162 @@
 Dask-Jobqueue
 =============
 
-*Easy deployment of Dask Distributed on job queuing systems like
-PBS, Slurm, and SGE.*
+*Easily deploy Dask on job queuing systems like PBS, Slurm, MOAB, and SGE.*
 
-Motivation
-----------
 
-1. While ``dask.distributed`` offers a flexible library for distributed computing
-   in Python, it is not always easy to deploy on systems that use job queuing
-   systems. Dask-jobqueue provides a Pythonic interface for deploying and
-   managing Dask clusters.
-2. In practice, deploying distributed requires customization, both for the
-   machine(s) that it will deployed on and for the specific application it will
-   be deployed for. Dask-jobqueue provides users with an intuitive interface for
-   customizing dask clusters.
+The Dask-jobqueue project makes it easy to deploy Dask on common job queuing
+systems typically found in high performance supercomputers, academic research
+institutions, and other clusters.  It provides a convenient interface that is
+accessible from interactive systems like Jupyter notebooks, or batch jobs.
 
 Example
 -------
 
 .. code-block:: python
 
    from dask_jobqueue import PBSCluster
+   cluster = PBSCluster()
+   cluster.scale(10)         # Ask for ten worker nodes
 
-   cluster = PBSCluster(processes=6, threads=4, memory="16GB")
-   cluster.start_workers(10)
+   # wait for jobs to arrive, depending on the queue, this may take some time
 
    from dask.distributed import Client
-   client = Client(cluster)
+   client = Client(cluster)  # Connect Dask to the worker nodes
 
-See :doc:`Examples <examples>` for more real-world examples.
+   import dask.array as da
+   x = ...                   # Dask commands now use these distributed resources
 
 
 Adaptivity
 ----------
 
-This can also adapt the cluster size dynamically based on current load.
-This helps to scale up the cluster when necessary but scale it down and save
-resources when not actively computing.
+Dask jobqueue can also adapt the cluster size dynamically based on current
+load.  This helps to scale up the cluster when necessary but scale it down and
+save resources when not actively computing.
 
 .. code-block:: python
 
    cluster.adapt(minimum=1, maximum=100)
 
+
+Configuration
+-------------
+
+Dask-jobqueue should be configured to your cluster so that it knows how many
+resources to request of each node and how to break up those resources.  You can
```

Comment:
Again the node vs job point of view. Not sure what we should say. At least mention the term job somewhere?

Reply:
Fixed